### PR TITLE
Create Boston skeleton

### PR DESCRIPTION
**Pull request by cobrien-dev**
This Pull request closes issue #41 Tracking Progress.

What should I add more categories?

### CODE REVIEW

File: cobrien-dev/cobrien-dev-Boston.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Boston
+
+#### Restaurants
+
```
Comment (guramipal) [inline cobrien-dev/cobrien-dev-Boston.md:9]: Legal Seafood.

**Comment by chanlon27** [comment]
testing a comment

**Comment by TomBurg** [comment]
I went to MIT in late 70's and loved Anthony's Pier 4
I going to Boston this Thursday/Friday - What other seafood places besides Legal near harbour?

**Comment by cobrien-dev** [comment]
:ship:

**Pull request merged**